### PR TITLE
Run at log on

### PR DESCRIPTION
**Pull request by Brummell67**

### CODE REVIEW

**Comment by Brummell67** [comment]
Sorry, I've some problems with my branch and maybe I've pushed bad. The code should be fixed.
I wait.

**Comment by dremin** [comment]
I still see a couple things not addressed, did you see them?

**Comment by Brummell67** [comment]
Try please

File: Cairo Desktop/CairoDesktop.Configuration/Properties/Settings.settings [hunk 1]
```diff
@@ -98,5 +98,8 @@
     <Setting Name="FullWidthTaskBar" Type="System.Boolean" Scope="User">
       <Value Profile="(Default)">False</Value>
     </Setting>
+    <Setting Name="RunAtLogOn" Type="System.Boolean" Scope="User">
```
Comment (dremin) [inline Cairo Desktop/CairoDesktop.Configuration/Properties/Settings.settings]: This is the last place the old setting is referenced. You should be able to remove this by going to the CairoDesktop.Configuration properties, then choosing Settings, and removing this setting.

**Comment by dremin** [comment]
Just one last item and it will be perfect!

**Comment by Brummell67** [comment]
I hope this version is correct.
I'm so sorry, I have to study version control better

**Comment by dremin** [comment]
Looks good, thanks!

**Pull request merged**